<ui version="4.0" >
 <class>Login</class>
 <widget class="QWidget" name="Login" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QPushButton" name="pushButton_Profile" >
     <property name="text" >
      <string>Profile</string>
     </property>
     <property name="icon" >
      <iconset resource="image.qrc" >
       <normaloff>:/profile/images/profiles.png</normaloff>:/profile/images/profiles.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>User</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_User" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_Password" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip" >
      <string/>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_Host" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_Port" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Database</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_DB" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Connect" >
     <property name="text" >
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_About" >
     <property name="text" >
      <string>About</string>
     </property>
     <property name="icon" >
      <iconset resource="image.qrc" >
       <normaloff>:/about/images/about.png</normaloff>:/about/images/about.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>231</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Hide" >
     <property name="text" >
      <string>&lt;&lt;&lt; Hide</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc" />
 </resources>
 <connections/>
</ui>
